src/main.ts

```diff
@@ -44,8 +46,11 @@ class LocalTableland {
 
     if (typeof config.validatorDir === "string")
       this.validatorDir = config.validatorDir;
-    if (typeof config.registryDir === "string")
+    if (typeof config.registryDir === "string" && config.registryDir) {
       this.registryDir = config.registryDir;
+    } else {
+      this.registryDir = defaultRegistryDir();
```

Comment:
If a path to an evm-tableland wasn't provided we use the default in this repo's registry directory